docs/docs/integrations/libraries/dbt/dbt-migration-components.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Migrating to dbt components
+sidebar_label: Migrating to dbt components
+description: How to migrate a Dagster dbt project from Pythonic to components.
+sidebar_position: 200
+---
+
+Dagster supports two main ways to integrate with dbt, [components](/integrations/libraries/dbt) and the creating assets with the [Pythonic API](/integrations/libraries/dbt/dbt-pythonic). If you have already built your Dagster and dbt project in the Pythonic way, you can migrate to using components and get the same result.
+
+## 1. Scaffold the dbt component
+
+The first step is to [scaffold the dbt component](/integrations/libraries/dbt#3-scaffold-a-dbt-component-definition). This will generate the `defs.yaml` configuration file with a path to your dbt project:
+
+<CodeExample
+  path="docs_snippets/docs_snippets/guides/components/integrations/dbt-component/7-component.yaml"
+  title="my_project/defs/dbt_ingest/defs.yaml"
+  language="yaml"
+/>
+
+## 2. Remove Pythonic definitions
+
+The component handles the creation of any dbt assets in your Dagster project as well as configuring the underlying resource. You can now remove the explicit dbt resource creation:
+
+<CodeExample
```

Comment:
Is this example showing code that should be deleted? If so, you may want to either highlight the lines to be deleted and add a comment to the example ("These lines can be removed") or add a tabbed section with before/after tabs.